web/frontend/src/mocks/handlers.ts

```diff
@@ -30,6 +30,16 @@ const { mockElections, mockResults } = setupMockElection();
 
 var mockUserDB = setupMockUserDB();
 
+const checkUserRole = (roles: ROLE[]): boolean => {
```

Comment:
very tiny remark 😄, otherwise LGTM!

```suggestion
const isAuthorized = (roles: ROLE[]): boolean => {
```